mne/io/ctf/tests/test_ctf.py

```diff
@@ -350,12 +351,13 @@ def test_read_ctf_annotations():
 
     raw = RawArray(
         data=np.empty((1, 432000), dtype=np.float64),
-        info=create_info(ch_names=1, sfreq=1200.0)
-    ).set_annotations(read_annotations(somato_fname))
+        info=create_info(ch_names=1, sfreq=1200.0))
+    raw.info['meas_date'] = read_raw_ctf(somato_fname).info['meas_date']
+    raw.set_annotations(read_annotations(somato_fname))
 
     events, _ = events_from_annotations(raw)
     latencies = np.sort(events[:, 0])
-    assert_array_equal(latencies, EXPECTED_LATENCIES)
+    assert_allclose(latencies, EXPECTED_LATENCIES, atol=1e-6)
```

Comment:
why?

Reply:
Same as above I think, we were operating at the precision limit before. Rather than change all values in `EXPECTED_LATENCIES` I just made it tolerant of a 1 μs error.